stan/math/prim/scal/fun/binomial_coefficient_log.hpp

```diff
@@ -68,12 +68,14 @@ namespace stan {
       if ((N < cutoff) || (N - n < cutoff)) {
         return lgamma(N + 1.0) - lgamma(n + 1.0) - lgamma(N - n + 1.0);
       } else {
-        return n * log(N - n) + (N + 0.5) * log(N/(N-n))
-          + 1/(12*N) - n - 1/(12*(N-n)) - lgamma(n + 1.0);
+        typename return_type<T_N, T_n>::type N_m_n(N - n);
```

Comment:
Yeah, I see it. I just broke up the `log(N - n)` into a separate call and this avoids the need to even do that.